examples/05-produce-cloud/src/main.rs

```diff
@@ -0,0 +1,47 @@
+//! A minimal example showing how to produce messages on Fluvio
```

Comment:
Maybe we can rephrase this to make it clear that it is using a profile different than the default. Maybe instead of produce-cloud we could name this as `produce-custom-profile` and then we say that we will use the `cloud` profile